python/openvino_tokenizers/convert_tokenizer.py

```diff
@@ -10,14 +10,47 @@
 from openvino.runtime.exceptions import OVTypeError
 
 from openvino_tokenizers.constants import UTF8ReplaceMode
-from openvino_tokenizers.utils import change_inputs_type, change_outputs_type, update_rt_info
-
+from openvino_tokenizers.utils import (
+    change_inputs_type,
+    change_outputs_type,
+    update_rt_info,
+    TokenzierConversionParams,
+)
 
 logger = logging.getLogger(__name__)
 
 
+def capture_arg(func):
+    def wrapper(*argc, **kwargs):
+        params = None
+        if len(argc) > 1 and argc[1] != None:
+            params = argc[1]
+        if 'params' in kwargs:
+            params = kwargs['params']
+        
+        if params is not None:
+            for key in TokenzierConversionParams.__match_args__:
+                if kwargs[key] is not None:
+                    msg = "Cannot specify both 'params' and individual convert_tokenizer arguments simultaneously. " \
+                          "Please pass all conversion params either individually, e.g. " \
+                          "convert_tokenizer(tokenizer_object, with_detokenizr=True, add_special_tokens=True,...). " \
+                          "Or within 'params' argument, e.g. " \
+                          "convert_tokenzier(tokenizer_object, params={'with_detokenizr': True, 'add_special_tokens': True, ...})"
+                    raise ValueError(msg)
+        
+        return func(*argc, **kwargs)
+    
+    # Embed convert_tokenizer docstring with TokenzierConversionParams docstring.
+    pos = func.__doc__.find('    Returns:')
+    wrapper.__doc__ = func.__doc__[:pos] + TokenzierConversionParams.__doc__ + '\n' + func.__doc__[pos:]
```

Comment:
https://docs.python.org/3/library/functools.html#functools.wraps

Reply:
Applied here @wraps and other stuff we discussed on teams.